server/src/index.ts

```diff
@@ -24,6 +24,17 @@ io.on("connection", (socket) => {
  console.log(`A user connected: ${socket.id}`);
 
  socket.on("join_room", (data) => {
+  
+  //Checking if there are any rooms currently connected to. If so looping through each room and users. If user connected to a room, leave the room. Also remove the user from the room sockets object
+  if(roomSockets){
```

Comment:
Is roomSockets a variable that was declared somewhere? Or is that something built into sockets.io?

Reply:
the roomSockets was the object you created in the previous pull request to keep a log of all connected users and rooms.

Reply:
Damn, you're right. I forgot about it because I haven't pulled the latest from main yet.